components/inputField/Input.tsx

```diff
@@ -5,14 +5,14 @@ export type InputProps = {
   type: string;
 };
 
-function UsernameInput({ placeholder, ...props }: InputProps) {
+function Input({ placeholder, ...props }: InputProps) {
   return (
     <input
-      className={`${styles.input} ${styles.usernameInput}`}
+      className={`${styles.input}`}
       placeholder={`${placeholder}`}
```

Comment:
```suggestion
      placeholder={`${placeholder}`}
```
is $ necessary here? or is the styles missing? because you used a pseudo-element